### ISSUE TITLE
npm run lint throws an error

### ISSUE THREAD

**Issue by webcoderkz** [nuxt/typescript#136]
**Describe the bug**
Added eslint typescript config, as per guide: https://typescript.nuxtjs.org/guide/lint.html
But now, when i run `npm run lint` i'm getting the following error:

```
 1 error and 0 warnings potentially fixable with the `--fix` option.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webcoder@1.0.0 lint: `eslint --ext .ts,.js,.vue .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webcoder@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**To Reproduce**
All the steps i did, is from this guide: https://typescript.nuxtjs.org/guide/lint.html

**Expected behavior**
It shouldn't throw ELIFECYCLE error

**Additional context**
Created project using `npx create-nuxt-app <project-name>` command.

**Comment by kevinmarrec** [comment]
Seems you don't have `eslint` installed, it may not be clear in the documentation as it assumes you have `eslint` installed only if you were using `@nuxtjs/eslint-config`

**Comment by kevinmarrec** [comment]
If it's not this problem, please a provide a repository of your current project, so I can quickly see where is the problem.

**Comment by webcoderkz** [comment]
@kevinmarrec eslint is actually installed. Here's the repo: https://gitlab.com/heihachi88/webcoder

**Comment by kevinmarrec** [comment]
@webcoderkz You need to remove https://gitlab.com/heihachi88/webcoder/blob/master/.eslintrc.js#L7
I need to tell it in documentation cause it's not trivial but : `@nuxtjs/eslint-config-typescript` set a TypeScript parser which is different from `babel-eslint`. If `babel-eslint` is kept here it overrides the TypeScript one, leading to bugs 😁

**Comment by kevinmarrec** [comment]
You may also need to remove https://gitlab.com/heihachi88/webcoder/blob/master/package.json#L22

And you can safely remove https://gitlab.com/heihachi88/webcoder/blob/master/package.json#L20 cause it's shipped by `@nuxtjs/eslint-config-typescript`

**Comment by webcoderkz** [comment]
Thank, would try it now.

**Comment by webcoderkz** [comment]
I've removed all the references you mentioned above, and now when i run `npm run dev`, i get the following error:

```
 ✖ Nuxt Fatal Error                                                                │
   │                                                                                     │
   │   Error: Cannot find module '@nuxtjs/eslint-module'                                 │
   │   Require stack:                                                                    │
   │   - /Users/username/Sites/webcoder/node_modules/@nuxt/core/dist/core.js         │
   │   - /Users/username/Sites/webcoder/node_modules/@nuxt/cli/dist/cli-command.js   │
   │   - /Users/username/Sites/webcoder/node_modules/@nuxt/cli/dist/cli.js           │
   │   - /Users/username/Sites/webcoder/node_modules/nuxt/bin/nuxt.js  
```

Tried to remove this module `@nuxtjs/eslint-module` but it throws an error.

**Comment by kevinmarrec** [comment]
Commit and push your changes I'll take a look

**Comment by webcoderkz** [comment]
Pushed rn.

**Comment by webcoderkz** [comment]
@kevinmarrec did you watch my repo?

**Comment by webcoderkz** [comment]
Seems like eslint typescript:

```
✖ 3 problems (3 errors, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webcoder@1.0.0 lint: `eslint --ext .ts,.js,.vue .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webcoder@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

By throwing these errors works as intended, because if i have no errors it finishes OK. Could you update documentation: https://typescript.nuxtjs.org/guide/lint.html

With the information regards removing:

```
parserOptions: {
  parser: 'babel-eslint'
},
```

**Comment by kevinmarrec** [comment]
@webcoderkz So you don't have issue anymore ? I'm gonna update docs right now

**Comment by kevinmarrec** [comment]
It's now live : https://typescript.nuxtjs.org/guide/lint.html

**Comment by webcoderkz** [comment]
@kevinmarrec yeah, seems like it's working as intended.
